libs/agno/agno/playground/router.py

```diff
@@ -104,52 +105,105 @@ def process_image(file: UploadFile) -> List[Union[str, Dict]]:
-            base64_image = process_image(body.image)
-
-        if body.stream:
+        if image:
+            base64_image = process_image(image)
+
+        if files:
+            for file in files:
+                if file.content_type == "application/pdf":
+                    contents = file.file.read()
+                    pdf_file = BytesIO(contents)
+                    pdf_file.name = file.filename
+                    file_content = PDFReader().read(pdf_file)
+                    if agent.knowledge is not None:
+                        agent.knowledge.load_documents(file_content)
+                elif file.content_type == "text/csv":
+                    contents = file.file.read()
+                    csv_file = BytesIO(contents)
+                    csv_file.name = file.filename
+                    file_content = CSVReader().read(csv_file)
+                    if agent.knowledge is not None:
+                        agent.knowledge.load_documents(file_content)
+                elif file.content_type == "application/vnd.openxmlformats-officedocument.wordprocessingml.document":
+                    contents = file.file.read()
+                    docx_file = BytesIO(contents)
+                    docx_file.name = file.filename
+                    file_content = DocxReader().read(docx_file)
+                    if agent.knowledge is not None:
+                        agent.knowledge.load_documents(file_content)
+                elif file.content_type == "text/plain":
+                    contents = file.file.read()
+                    text_file = BytesIO(contents)
+                    text_file.name = file.filename
+                    file_content = TextReader().read(text_file)
+                    if agent.knowledge is not None:
+                        agent.knowledge.load_documents(file_content)
+                else:
+                    raise HTTPException(status_code=400, detail="Unsupported file type")
+
+        if stream:
             return StreamingResponse(
-                chat_response_streamer(new_agent_instance, body.message, base64_image),
+                chat_response_streamer(new_agent_instance, message, images=base64_image),
                 media_type="text/event-stream",
             )
         else:
-            run_response = cast(RunResponse, new_agent_instance.run(body.message, images=base64_image, stream=False))
+            run_response = cast(
+                RunResponse,
+                new_agent_instance.run(
+                    message,
+                    images=base64_image,
+                    stream=False,
+                ),
+            )
             return json.dumps(asdict(run_response))
 
-    @playground_router.post("/agent/sessions/all")
-    def get_agent_sessions(body: AgentSessionsRequest):
-        logger.debug(f"AgentSessionsRequest: {body}")
-        agent = get_agent_by_id(body.agent_id, agents)
+    @playground_router.get("/agent/{agent_id}/user/{user_id}/session/all")
```

Comment:
```suggestion
    @playground_router.get("/agents/{agent_id}/sessions")
```
And then add a query parameter `?user_id={user_id}` to get the user ID.